cloudbuild_CI.yaml

```diff
@@ -0,0 +1,50 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
```

Comment:
How and Where is this file used?

Reply:
This is used when setting up a Build Trigger config for a project to monitor GitHub changes.
The comment below makes a reference to it.